sentry/src/routes/channel.rs

```diff
@@ -846,4 +973,156 @@ mod test {
             balances.spenders.get(&CREATOR),
         );
     }
+
+    #[tokio::test]
+    async fn payouts_for_earners_test() {
+        let app = setup_dummy_app().await;
+        let channel_context = app
+            .config
+            .find_chain_of(DUMMY_CAMPAIGN.channel.token)
+            .expect("Dummy channel Token should be present in config!")
+            .with(DUMMY_CAMPAIGN.channel);
+
+        insert_channel(&app.pool, channel_context.context)
+            .await
+            .expect("should insert channel");
+        insert_campaign(&app.pool, &DUMMY_CAMPAIGN)
+            .await
+            .expect("should insert the campaign");
+
+        // Setting the initial remaining to 0
+        let campaign_remaining = CampaignRemaining::new(app.redis.clone());
+        campaign_remaining
+            .set_initial(DUMMY_CAMPAIGN.id, UnifiedNum::from_u64(0))
+            .await
+            .expect("Should set value in redis");
+
+        let auth = Auth {
+            era: 0,
+            uid: ValidatorId::from(DUMMY_CAMPAIGN.creator),
+            chain: channel_context.chain.clone(),
+        };
+
+        let build_request = |channel_context: &ChainOf<Channel>, payouts: UnifiedMap| {
+            let payouts = ChannelPayRequest { payouts };
+
+            let body = Body::from(serde_json::to_string(&payouts).expect("Should serialize"));
+
+            Request::builder()
+                .extension(channel_context.clone())
+                .extension(auth.clone())
+                .body(body)
+                .expect("Should build Request")
+        };
+
```

Comment:
Could you also perform requests before the rest of the setup is set?

This will make sure that if Accounting or spendable is missing it will still function and return a sane response.